### ISSUE TITLE
Go.mod seems not updated correctly 

### ISSUE THREAD

**Issue by AndreaEsposit** [wasmerio/wasmer-go#174]
### Description
 Hey, I kinda just followed the basic example, but I keep getting an error when I try to install the library.  Instead of getting the v1.0.0 version, it keeps "redirecting" me to the v0.3.1 version. 

Plus in the example, it seems like you are importing the library as "wasm", but then you refer to it as wasmer right after. Is it a spelling error, or am I missing something?  (I am sorry if it is something obvious)
``` 
package main

import (
	"fmt"
	wasm "github.com/wasmerio/wasmer-go/wasmer"
)

func main() {
    wasmBytes, _ := wasm.ReadBytes("simple.wasm")

    engine := wasmer.NewEngine()
    store := wasmer.NewStore(engine)
.........
``` 

Using the latest commit hash works fine fortunately, but then it is unclear where the function  "ReadBytes()"  is from, are you using another name for it now?  

### Steps to reproduce
Try to install the library:
```go get github.com/wasmerio/wasmer-go/wasmer```

### Expected behavior
Normal installation of the library. 

### Actual behavior
``` 
> go get -v github.com/wasmerio/wasmer-go/wasmer
go: github.com/wasmerio/wasmer-go upgrade => v0.3.1
go get: github.com/wasmerio/wasmer-go@v0.3.1: parsing go.mod:
        module declares its path as: github.com/wasmerio/go-ext-wasm
                but was required as: github.com/wasmerio/wasmer-go 
```

**Comment by syrusakbary** [comment]
This is because we haven't published (yet) a stable 1.0.0 version.
We are working on it! Hopefully this issue will not be open for long :)

**Comment by Hywan** [comment]
1.0.0 released. I believe the problem is solved. Closing the issue. Feel free to re-open if it persists.
